web/src/pages/barcoding.js

```diff
@@ -1,9 +1,335 @@
-import React from 'react'
-
-export default function BarcodingPage() {
-    return (
-        <div>
-            <h1>this is the Barcoding Page</h1>
-        </div>
-    )
+import React from 'react';
+import tw from 'twin.macro';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+
+import Layout from '../components/Layout'
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
```

Comment:
I don't think this is doing anything. Can you confirm and remove if it's not?

Reply:
@asasmith it is. When we load the Layout component we have to adjust the image

Reply:
but you're also adding it to the `.gatsby-image-wrapper`? Do we need both? Adjusting the value in the dev tools doesn't seem to change anything.

Reply:
also the `width` and `${tw'w-screen'}` are maybe redundant.

Reply:
@asasmith Yeah I went back and tested it, I thought you were referring to the -160px! Ive removed both actually

Reply:
I don't think you line 10 `top: -160px`. When I adjust it in the dev tools nothing changes. 
It looks like the positioning is happening on lines 13-15.

Even if I remove the styles your applying to `.gatsby-image-wrapper` the `top: -160px` doesn't appear to do anything. Let's remove it and just use the styling applied to `.gatsby-image-wrapper` for now